packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/dedicatedCloud-datacenter-dashboard.html

```diff
@@ -185,6 +185,7 @@
                 service-name="{{ ::$ctrl.serviceName }}"
                 query-params="{
                     datacenterId: $ctrl.datacenter.model.id,
+                    scope: 'datacenter',
```

Comment:
I wonder if `service-type` should be something like `dedicated-cloud.datacenter`